osu.Game.Rulesets.Taiko/UI/IHitExplosion.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Objects.Drawables;
+
+namespace osu.Game.Rulesets.Taiko.UI
+{
+    /// <summary>
+    /// Interface for hit explosions shown on the playfield's hit target in taiko.
+    /// </summary>
+    public interface IHitExplosion
```

Comment:
This interface really confused me at first, becuase `HitExplosion` doesn't implement it. Maybe it needs a better name?

Reply:
Going to follow `IAnimatableJudgement`'s lead here and make this an `IAnimatableHitExplosion`. Let me know if that's better.